packages/manager/apps/telecom/src/app/telecom/telephony/line/softphone/addDevice/add-device.controller.js

```diff
@@ -0,0 +1,10 @@
+export default class SoftphoneAddDeviceController {
+  /* @ngInject */
```

Comment:
```suggestion
```
you don't have import in controller so no need

Reply:
I added coreUrlBuilder so required now ;)